### PR TITLE
Chatwork support

### PR DESCRIPTION
**Pull request by RVIRUS0817**
## What did you implement:

Closes #XXXXX

## Todos:
You don't have to satisfy all of the following.

- [✓ ] Write tests
- [✓ ] Write documentation
- [✓] Check that there aren't other open pull requests for the same issue/feature
- [✓] Format your source code by `make fmt`
- [✓] Pass the test by `make test`
- [✓] Provide verification config / commands
- [✓] Enable "Allow edits from maintainers" for this PR
- [✓] Update the messages below

***Is this ready for review?:*** NO  
***Is it a breaking change?:*** NO

**・document**
https://github.com/vulsdoc/vuls/pull/26

**・test report done**

### CODE REVIEW

File: report/chatwork.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+	payload := url.Values{
+		"body": {message},
+	}
+	reqs, err := http.NewRequest("POST", uri, strings.NewReader(payload.Encode()))
+
+	reqs.Header.Add("X-ChatWorkToken", token)
+	reqs.Header.Add("Content-Type", "application/x-www-form-urlencoded")
+
+	if err != nil {
```
Comment (kotakanbe) [inline report/chatwork.go]: move to L54
https://github.com/future-architect/vuls/pull/634/files#diff-02cee48536ff2ce0d7263f3b702cb4a1R53

File: report/chatwork.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+// ChatWorkWriter send report to ChatWork
+type ChatWorkWriter struct{}
+
+func (w ChatWorkWriter) Write(rs ...models.ScanResult) (err error) {
+	conf := config.Conf.ChatWork
+
+	var message string
+	for _, r := range rs {
+		serverInfo := fmt.Sprintf("%s", r.ServerInfo())
+		err = ChatWorkpostMessage(conf.Room, conf.ApiToken, serverInfo)
```
Comment (kotakanbe) [inline report/chatwork.go]: if err = ChatWorkpostMessage(conf.Room, conf.ApiToken, serverInfo); err != nil {
   return err
}

File: report/chatwork.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+		}
+
+		for _, vinfo := range r.ScannedCves {
+			maxCvss := vinfo.MaxCvssScore()
+			severity := strings.ToUpper(maxCvss.Value.Severity)
+			if severity == "" {
+				severity = "?"
+			}
+
+			message = serverInfo + "[info]" + "[title]" + `https://nvd.nist.gov/vuln/detail/` + vinfo.CveID + "  " + strconv.FormatFloat(maxCvss.Value.Score, 'f', 1, 64) + " " + "(" + severity + ")" + "[/title]" + vinfo.Summaries(config.Conf.Lang, r.Family)[0].Value + "[/info]"
```
Comment (kotakanbe) [inline report/chatwork.go]: use fmt.Sprintf

File: report/chatwork.go [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+			err = ChatWorkpostMessage(conf.Room, conf.ApiToken, message)
```
Comment (kotakanbe) [inline report/chatwork.go]: if err = ChatWorkpostMessage(conf.Room, conf.ApiToken, message); err != nil {
  return err
}

File: report/chatwork.go (hunk 1 above)
Comment (RVIRUS0817) [reply report/chatwork.go]: noting error thanks!

File: report/chatwork.go (hunk 2 above)
Comment (RVIRUS0817) [reply report/chatwork.go]: noting error thanks!(part2)

File: report/chatwork.go (hunk 4 above)
Comment (RVIRUS0817) [reply report/chatwork.go]: noting error thanks!(part3)

File: report/chatwork.go (hunk 3 above)
Comment (RVIRUS0817) [reply report/chatwork.go]: I fixed ! Thank u so much!

**Pull request merged**